rpcd/playbooks/roles/logstash/tasks/logstash_web_disable.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: prevent logstash-web from starting on boot
+  copy: 
+    dest: "/etc/init/logstash-web.override"
+    content: "manual\n"
+    owner: root
+    group: root
+    mode: 644
+  tags:
+    - logstash-web-disable
+
+- name: stop logstash-web
+  service: 
+    name: logstash-web
+    state: stopped
+    enabled: no
+  tags:
+    - logstash-web-disable
```

Comment:
no race.  
osad-integration already lost or this would have been merged more than an hour ago